_posts/wasmer-go-embedding-1.0.md

```diff
@@ -0,0 +1,543 @@
+---
+title: 'Wasmer Go embedding 1.0 liftoff'
+excerpt: 'Announcing the immediate availability of the Wasmer Go embedding 1.0 version!'
+date: '2021-02-22T15:35:07.322Z'
+author: Ivan Enderlin
+---
+
+We are delighted to announce the availability of the [Wasmer Go
+embedding 1.0 version](https://github.com/wasmerio/wasmer-go).
+
+A little more than 1.5 years, we first release `wasmer-go`, the Wasmer
+embedding for Go. The reception by the community was beyond our
+expectations. Back then, it was [the fastest WebAssembly runtime for
+Go](https://medium.com/wasmer/announcing-the-fastest-webassembly-runtime-for-go-wasmer-19832d77c050).
```

Comment:
We should rephrase this statement. Reading this as it is, it implies Wasmer that is no longer the fastest WebAssembly runtime for go.

Something like:
```suggestion
expectations, the repercussion of launching [the fastest WebAssembly runtime for
Go](https://medium.com/wasmer/announcing-the-fastest-webassembly-runtime-for-go-wasmer-19832d77c050) was a great way to start.
```

Reply:
:+1: